src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -486,16 +533,21 @@ export class NfsFormComponent extends CdForm implements OnInit {
   }
 
   private generatePseudo() {
-    let newPseudo = this.nfsForm.getValue('pseudo');
-    if (this.nfsForm.get('pseudo') && !this.nfsForm.get('pseudo').dirty) {
-      newPseudo = undefined;
-      if (this.storageBackend === 'CEPH') {
-        newPseudo = '/cephfs';
-        if (_.isString(this.nfsForm.getValue('path'))) {
-          newPseudo += this.nfsForm.getValue('path');
-        }
+    const pseudoControl = this.nfsForm.get('pseudo');
+    let newPseudo =
+      pseudoControl && pseudoControl.dirty ? this.nfsForm.getValue('pseudo') : undefined;
+
+    if (!newPseudo) {
+      const path = this.nfsForm.getValue('path');
+      newPseudo = this.storageBackend === 'CEPH' ? '/cephfs' : '/rgw';
```

Comment:
There is an exisiting function for this `getPathfromFsal`
```suggestion
      newPseudo = getPathfromFsal(this.storageBackend);
```

Reply:
I see `getPathfromFsal` return `cephfs` or `rgw`